test/liquidityBridgeContract.js

```diff
@@ -17,15 +17,114 @@ contract('LiquidityBridgeContract', async accounts => {
         let currAddr = web3.eth.currentProvider.getAddress();
         let existing = await instance.getCollateral(currAddr); 
 
-        await instance.register({value : val});
+        await instance.register({value : val, from: web3.eth.currentProvider.addresses[0]});
```

Comment:
Isn't this the default behavior? Also, could use lpRskAddress (I think) variable here instead.

Reply:
removed param.